lib/flipper/gates/percentage_of_actors.rb

```diff
@@ -30,7 +30,7 @@ def open?(context)
         if Types::Actor.wrappable?(context.thing)
           actor = Types::Actor.wrap(context.thing)
           id = "#{context.feature_name}#{actor.value}"
-          Zlib.crc32(id) % 100 < percentage
+          Zlib.crc32(id) % 100_000 < percentage * 1_000
```

Comment:
Tweaked in bba3969f8aff09717ec007a2adcbcb530c237f62. Do you think that helps? Enough? Any other suggestions?